nuxt.config.ts

```diff
@@ -13,6 +12,12 @@ export default defineNuxtConfig({
     ],
     langDir: './locales',
     defaultLocale: 'en'
+  },
+  css: ['~/assets/css/main.css'],
+  postcss: {
+    plugins: {
+      tailwindcss: {},
+      autoprefixer: {}
+    }
```

Comment:
I think it would be better to set 'nesting' as well https://tailwindcss.com/docs/using-with-preprocessors#nesting

Reply:
You mean to add an empty default value, like `'tailwindcss/nesting': {},` or you want to override it with an alternative thing, like `'tailwindcss/nesting': 'postcss-nesting',`?

Reply:
I mean empty default value 'tailwindcss/nesting': {}. as I remember it gives some warning otherwise not set.

Reply:
I think we have no warnings here (yet?). I followed this guide: https://tailwindcss.com/docs/guides/nuxtjs
But sure, if you think it makes sense, let it be. Added :)

Reply:
@sumerokr sorry for the confusion. Gives warning if you try to use nesting.